pkg/storer/reserve.go

```diff
@@ -28,8 +28,13 @@ const (
 	reserveUnreserved   = "reserveUnreserved"
 	batchExpiry         = "batchExpiry"
 	batchExpiryDone     = "batchExpiryDone"
+
+	MaxReserveCapacityHeight = 1
```

Comment:
reserve does not need to know about maxHeight

Reply:
fixed